signup page added

lawyerturtle6 opened the pull request.

lawyerturtle6 committed:
signup page added

ldfMion reviewed (changes requested):
Hi, it just needs some changes
- Use ShadCn components for cards, inputs, buttons, labels, etc
- add logic in the account page to show the user isn't logged in
- merge dev into your branch

lawyerturtle6 committed:
Merge branch 'dev' into jakef

lawyerturtle6 committed:
fixed signup page to use shadcn components,
modified account page to show current logged in account,
login button becomes switch accounts if logged in

ldfMion committed:
adapt to shadcn and existing project style

ldfMion reviewed (approved):
added shadcn card component, removed unnecessary styling, and merged
